From b99d192eddd4d94dcb1eb0eb4b3693e7c76cea99 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Fri, 22 Nov 2019 11:39:14 -0600
Subject: [PATCH 21/34] ARM64: dts: logicpd-imx8mm-kit: Add support for
 pcf85263 RTC

A pcf85263 is on the SOM connected to I2C3 for use as the primary
RTC device and is powered by a backup battery on the baseboard.

This patch patch adds the functionality to the logicpd-imx8mm-kit.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi b/arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi
index 485e3d6aa2a7..869d74f2a7de 100644
--- a/arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/lpd-imx8mm-som.dtsi
@@ -190,6 +190,11 @@
 		read-only;	/* Manufacturing EEPROM programmed at factory */
 		reg = <0x50>;
 	};
+
+	rtc@51 {
+		compatible = "nxp,pcf85263";
+		reg = <0x51>;
+	};
 };
 
 &iomuxc {
-- 
2.17.1

